package.json

```diff
@@ -88,7 +88,7 @@
     "@gnosis.pm/safe-apps-sdk-v1": "npm:@gnosis.pm/safe-apps-sdk@0.4.2",
     "@gnosis.pm/safe-core-sdk": "^1.3.0",
     "@gnosis.pm/safe-deployments": "^1.8.0",
-    "@gnosis.pm/safe-react-components": "^0.9.0",
+    "@gnosis.pm/safe-react-components": "https://github.com/gnosis/safe-react-components#add-gift-icon",
```

Comment:
Make sure to update this to the release before merging.

Reply:
Bumped the version to 0.9.8